helpers/components_vanila/animations/parallax/parallax.js

```diff
@@ -0,0 +1,34 @@
+import gsap from 'gsap';
+import { ScrollTrigger } from 'gsap/ScrollTrigger';
+
+const parallaxInit = () => {
+	gsap.registerPlugin(ScrollTrigger);
+
+	const $parallaxItems = document.querySelectorAll('.js-parallax-item');
+
+	if (!$parallaxItems.length) return;
```

Comment:
`
if (!$parallaxItems.length) {
   console.error('Parallax elements not found!');
   return;
};
`